src/http/sr-modules-keys/sr-modules-keys.controller.ts

```diff
@@ -41,7 +41,7 @@ export class SRModulesKeysController {
         reply.type('application/json').send(jsonStream);
 
         for (const { keysGenerator, module } of keysGeneratorsByModules) {
-          // TODO: does memory consumption increased ?
+          // TODO: does memory consumption is increase ?
           const keysData = { module, keys: [] as any[] };
```

Comment:
can we add more strict types?

Reply:
returned prev solution as here streams doesnt work properly

Reply:
and added types